esmvalcore/preprocessor/_regrid.py

```diff
@@ -383,6 +383,18 @@ def extract_location(cube, location, scheme):
                          scheme)
 
 
+def _check_grid_discontiguities(cube, scheme):
+    """Check if there are grid discontiguities and set use_src_mask to True."""
+    try:
+        discontiguities = iris.util.find_discontiguities(cube)
+    except NotImplementedError:
+        pass
+    else:
+        if np.isin(True, discontiguities):
```

Comment:
```suggestion
        if discontiguities.any():
```
would look a bit more descriptive to me, but the current implementation is also fine.